io/FV3GFS_io.F90

```diff
@@ -1077,16 +1077,16 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain)
 
         if (Sfcprop(nb)%lakefrac(ix) > zero) then
           Sfcprop(nb)%oceanfrac(ix) = zero ! lake & ocean don't coexist in a cell
-          if (Sfcprop(nb)%fice(ix) < Model%min_lakeice) then
-             Sfcprop(nb)%fice(ix) = zero
-             if (Sfcprop(nb)%slmsk(ix) == 2) Sfcprop(nb)%slmsk(ix) = 0
-          endif
+!          if (Sfcprop(nb)%fice(ix) < Model%min_lakeice) then
```

Comment:
May I ask if these statements commented out are required for b4 restart reproducibility? I am curious why this change is required as it might indicate the fice could be <min_lakeice in restart run.

Reply:
I guess this question is for Dom.
(this code in my branch is slightly different; it never made it  to the dev version).

Reply:
I copied over the exact changes that were made in PRs #178 and #212 for develop. I don't think we should try to do something different here than what was done for develop (and what took us - well, actually Moorthi - months to figure out when we finalized the GFS v16 prod equivalent in develop). Please see these two PRs.

Reply:
> I guess this question is for Dom.
> (this code in my branch is slightly different; it never made it to the dev version).

Forgot to say, the code changes in this PR are from you originally - as you said, your slightly different version never made it to develop at that time, I remember those back and forth conversations quite well.